proto/executor.proto

```diff
@@ -91,6 +91,7 @@ message TableScan {
 	repeated KeyRange ranges = 6 [(gogoproto.nullable) = false]; // For global read in join, we must point out the key ranges when we don't have the region info.
 	repeated int64 primary_prefix_column_ids = 7;
 	optional bool keep_order = 8;
+	optional bool is_fast_scan = 9;
```

Comment:
Prefer to add a comment that this fast_scan is a feature only provided by TiFlash (but not TiKV).